lib/c/criu.c

```diff
@@ -1867,8 +1867,8 @@ void criu_set_pidfd_store_sk(int sk)
 
 int criu_local_set_network_lock(criu_opts *opts, enum criu_network_lock_method method)
 {
-	opts->rpc->has_network_lock = true;
 	if (method == CRIU_NETWORK_LOCK_IPTABLES || method == CRIU_NETWORK_LOCK_NFTABLES || method == CRIU_NETWORK_LOCK_SKIP) {
+		opts->rpc->has_network_lock = true;
```

Comment:
dropping an (unrelated) comment here, because I can't on line 1875.

Some other APIs return a -errno return code on errors, does the `-1` here must be treated differently (I guess it doesn't mean `EPERM`)?  Should it be changed to `-EINVAL`?

Reply:
Good point. Not sure if this, however, would count as an API break. It isn't very consistent, that is true.